cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -75,14 +76,18 @@ def _cleanup():
         request_details_list.cancel_migration(vm, confirmed=True)
 
     conversion_host_popup = []
-    host_creds = provider.hosts.all()
-    hosts_dict = {key.name: [] for key in host_creds}
+    if provider.one_of(RHEVMProvider):
+        host_creds = [h.name for h in provider.hosts.all()]
```

Comment:
Can you please elaborate on why the variable is called `host_creds` when it is storing `host_name` ?